src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -87,10 +87,34 @@ public static void TestResolveInvalidPaths()
 
         [Fact]
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap |    //[ActiveIssue(13121)]  // Access to path is denied in UWP
-        TargetFrameworkMonikers.NetFramework)]    // Full framework uses XmlUrlResolver instead of SystemPathResolver (which doesn't allow access to this path)
+        TargetFrameworkMonikers.NetFramework)]    // Full framework uses WebRequest instead of HttpRequest
         public static void TestResolveInvalidPath()
         {
-            AssertInvalidPath("ftp://www.bing.com");
+            Assert.Throws<System.ArgumentException>(() => XmlReader.Create("ftp://www.bing.com")); // Only 'http' and 'https' schemes are allowed.
```

Comment:
Correction: Looks like we actually do have tests for http and https:
- [ReadAsyncAfterInitializationWithUriThrows](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/tests/XmlReader/Tests/AsyncReaderLateInitTests.cs#L72)
- [ReadAfterInitializationWithUriOnAsyncReaderTrows](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/tests/XmlReader/Tests/AsyncReaderLateInitTests.cs#L83)